packages/graphql/src/schema/scalars/DateTime.ts

```diff
@@ -17,16 +17,28 @@
  * limitations under the License.
  */
 
-import { GraphQLScalarType } from "graphql";
-import neo4j from "neo4j-driver";
+import { GraphQLError, GraphQLScalarType } from "graphql";
+import neo4j, { DateTime, isDateTime } from "neo4j-driver";
```

Comment:
One of two points of contention here as unsure if this will work with older driver versions. If not, we need to either update our minimum, or find an alternative route.

Reply:
I'm curious about why it would not work?

Reply:
I believe these exports were only added in more recent release of the driver, hence why we did the weird `neo4j.types.DateTime` thing.